frontend/src/components/ReportSelector/reportSelectorConfig.tsx

```diff
@@ -409,6 +409,14 @@ export const categoriesAndTypes: IReportCategoriesAndTypes = [
         exportPDF: false,
         exportXLSX: true,
       },
+      {
+        value: "Tab_8_rpt_CA_ContractsbySubcontractor",
+        label: "Contract History for Specific Subcontractor Team",
+        description: `ipsem lorum`,
```

Comment:
```suggestion
        description: `Sorted by Fiscal, Shows CO#, Amendment# project #, ContractDescription,StartDate, EndDate, Contract Value Used, Status`,
```

Reply:
Thanks, I wasn't able to access the shared folder with descriptions for some reason.